docs/Write-Plugin.md

```diff
@@ -0,0 +1,58 @@
+<h1 align="center">Fastify</h1>
+
+# How to write a good plugin
+First of all thank you for having decided to build a plugin for Fastify. Fastify is a *battery not included* framework and plugins are its strength, so thank you.<br>
```

Comment:
"First of all thank you for having decided to build a plugin for Fastify." => "First, thank you for deciding to write a plugin for Fastify."

"Fastify is a *battery not included* framework" => "Fastify is a *batteries not included* framework"